<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fdm_toolbox</class>
 <widget class="QWidget" name="fdm_toolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fiber Distribution Map Toolbox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QLineEdit" name="line_edit_offset_y">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_offset">
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_samples">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Samples</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="line_edit_size_y">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>32</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="line_edit_size_z">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="line_edit_offset_z">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_size">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="line_edit_size_x">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>32</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="line_edit_offset_x">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QSlider" name="slider_clustering_threshold">
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="sliderPosition">
      <number>500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="checkbox_clustering_enabled">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="line_edit_samples_x">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>48</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="line_edit_samples_y">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>48</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_clustering_enabled">
     <property name="text">
      <string>Clustering</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_depths">
     <property name="text">
      <string>Layers</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_threshold">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Cluster Threshold</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="checkbox_auto_update">
     <property name="text">
      <string>Auto update</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>96</width>
       <height>67</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QPushButton" name="button_update">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkbox_enabled">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_enabled">
     <property name="text">
      <string>Enabled</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <layout class="QGridLayout" name="layout_depth_checkboxes">
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkbox_depth_5">
       <property name="text">
        <string>5</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="checkbox_depth_7">
       <property name="text">
        <string>7</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="checkbox_depth_4">
       <property name="text">
        <string>4</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="checkbox_depth_8">
       <property name="text">
        <string>8</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="checkbox_depth_3">
       <property name="text">
        <string>3</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkbox_depth_6">
       <property name="text">
        <string>6</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkbox_depth_0">
       <property name="text">
        <string>0</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checkbox_depth_2">
       <property name="text">
        <string>2</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkbox_depth_1">
       <property name="text">
        <string>1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="checkbox_depth_9">
       <property name="text">
        <string>9</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkbox_enabled</tabstop>
  <tabstop>line_edit_offset_x</tabstop>
  <tabstop>line_edit_offset_y</tabstop>
  <tabstop>line_edit_offset_z</tabstop>
  <tabstop>line_edit_size_x</tabstop>
  <tabstop>line_edit_size_y</tabstop>
  <tabstop>line_edit_size_z</tabstop>
  <tabstop>line_edit_samples_x</tabstop>
  <tabstop>line_edit_samples_y</tabstop>
  <tabstop>checkbox_depth_0</tabstop>
  <tabstop>checkbox_depth_1</tabstop>
  <tabstop>checkbox_depth_2</tabstop>
  <tabstop>checkbox_depth_3</tabstop>
  <tabstop>checkbox_depth_4</tabstop>
  <tabstop>checkbox_depth_5</tabstop>
  <tabstop>checkbox_depth_6</tabstop>
  <tabstop>checkbox_depth_7</tabstop>
  <tabstop>checkbox_depth_8</tabstop>
  <tabstop>checkbox_depth_9</tabstop>
  <tabstop>checkbox_clustering_enabled</tabstop>
  <tabstop>slider_clustering_threshold</tabstop>
  <tabstop>checkbox_auto_update</tabstop>
  <tabstop>button_update</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
